CHANGELOG.rst

```diff
@@ -11,6 +11,15 @@ Minor
 * Remove dependence on all ``source_stream`` APIs except for ``read()``.
   `#103 <https://github.com/aws/aws-encryption-sdk-python/issues/103>`_
 
+Potentially Breaking
+^^^^^^^^^^^^^^^^^^^^
+
+* Encryption streams no longer close the ``source_stream`` when they themselves close.
+  If you are using context managers for all of your stream handling,
+  this change will not affect you.
+  However, if you have been relying on this to close your ``source_stream`` for you,
```

Comment:
Would you mind expanding "this" into a class name or at least a more specific reference, to try to be as pedantically clear as possible to someone reading this quickly?

Reply:
done